package.json

```diff
@@ -32,5 +32,8 @@
     "unfetch": "latest",
     "vue": "latest",
     "vue-template-compiler": "latest"
+  },
+  "dependencies": {
+    "terminal-link": "^2.1.1"
```

Comment:
Should be a devDependency in `packages/loading-screen/package.json`

Reply:
BTW nice find :heart_eyes:  We should use it for other places.

Reply:
thanks. yes it's pretty nice and simple to use